config/settings/stupoli.py

```diff
@@ -0,0 +1,26 @@
+# Custom settings for stupoli deploy
+# -----------------------------------------------------------------------------
+
+OSLER_ROLE_DASHBOARDS = {
+    'Attending': 'dashboard-attending',
+    'Physician': 'dashboard-attending',
+}
+
+OSLER_DISPLAY_REFERRALS = False
+OSLER_DISPLAY_APPOINTMENTS = False
+OSLER_DISPLAY_CASE_MANAGERS = False
+OSLER_DISPLAY_ATTESTABLE_BASIC_NOTE = False
+OSLER_DISPLAY_DIAGNOSIS = False
+OSLER_DISPLAY_VOUCHERS = False
+OSLER_DISPLAY_WILL_RETURN = False
+OSLER_DISPLAY_ATTENDANCE = True
+OSLER_DISPLAY_FOLLOWUP = False
+OSLER_DISPLAY_VACCINE = False
+
+OSLER_DEFAULT_CITY = "Hamburg"
+OSLER_DEFAULT_STATE = "Hamburg"
+OSLER_DEFAULT_ZIP_CODE = "20459"
+OSLER_DEFAULT_COUNTRY = "Deutschland"
+OSLER_DEFAULT_ADDRESS = "Seewartenstraße 10"
+
+OSLER_ABOUT_NAME = "wiki.stupoli-hamburg.de"
```

Comment:
The about name should just be a name, not the actual url.

You should add an OSLER_ABOUT_URL setting, making its default the GitHub url, then overriding to your wikipedia page for StuPoli and the sojourner site that was previously linked for UMKC.

Reply:
I've added a OSLER_ABOUT_URL and set it to the GitHub url in base.py, thanks for the hint!